nixos/modules/services/mail/postfix.nix

```
@@ -584,6 +514,163 @@ in
               ])
             );
           options = {
+            message_size_limit = mkOption {
+              type = with types; nullOr int;
+              default = 10240000;
+              example = 52428800; # 50 MiB
+              description = ''
+                Maximum size of an email message in bytes.
+
+                <https://www.postfix.org/postconf.5.html#message_size_limit>
+              '';
+            };
+
+            mydestination = mkOption {
+              type =
+                with types;
+                nullOr (oneOf [
+                  str
+                  (listOf str)
+                ]);
+              default = [
+                "$myhostname"
+                "localhost.$mydomain"
+                "localhost"
+              ];
+              description = ''
+                List of domain names intended for local delivery using /etc/passwd and /etc/aliases.
+
+                ::: {.warning}
+                Do not include [virtual](https://www.postfix.org/VIRTUAL_README.html) domains in this list.
+                :::
+
+                <https://www.postfix.org/postconf.5.html#mydestination>
+              '';
+            };
+
+            myhostname = mkOption {
+              type = with types; nullOr types.str;
+              default = null;
+              example = "mail.example.com";
+              description = ''
+                The internet hostname of this mail system.
+
+                <https://www.postfix.org/postconf.5.html#myhostname>
+              '';
+            };
+
+            mynetworks = mkOption {
+              type = with types; nullOr (listOf str);
+              default = null;
+              example = [
+                "127.0.0.0/8"
+                "::1"
+              ];
+              description = ''
+                List of trusted remote SMTP clients, that are allowed to relay mail.
+
+                Leave unset to let Postfix populate this list based on the {option}`mynetworks_style` setting.
+
+                <https://www.postfix.org/postconf.5.html#mynetworks>
+              '';
+            };
+
+            mynetworks_style = mkOption {
+              type =
+                with types;
+                nullOr (enum [
+                  "host"
+                  "subnet"
+                  "class"
+                ]);
+              default = "host";
+              description = ''
+                The method used for generating the default value for {option}`mynetworks`, if that option is unset.
+
+                <https://www.postfix.org/postconf.5.html#mynetworks_style>
+              '';
+            };
+
+            recipient_delimiter = lib.mkOption {
+              type = with types; nullOr str;
+              default = null;
```

```suggestion
              default = "";
```
if I understand https://www.postfix.org/postconf.5.html#recipient_delimiter correctly.